upstream/jobs.go

```diff
@@ -134,15 +138,33 @@ func reconcileTable(ctx context.Context, config UpstreamConfig, table pushableTa
 
 		count += len(items)
 
-		if c, ok := table.(customIsPushedUpdater); ok {
-			if err := c.UpdateIsPushed(ctx.DB(), items); err != nil {
-				return 0, fmt.Errorf("failed to update is_pushed for %s: %w", table.TableName(), err)
-			}
-		} else {
-			ids := lo.Map(items, func(a models.DBTable, _ int) string { return a.PK() })
-			if err := ctx.DB().Model(table).Where("id IN ?", ids).Update("is_pushed", true).Error; err != nil {
-				return 0, fmt.Errorf("failed to update is_pushed on %s: %w", table.TableName(), err)
+		backoff := retry.WithMaxRetries(3, retry.NewConstant(time.Second))
```

Comment:
```suggestion
       backoff = time.Second
       if len(items) > 500:
            backoff = 3 * time.Second
       elseif len(items) > 100:
               backoff = 2 * time.Second
        
		backoff := retry.WithMaxRetries(3, retry.NewExponential(backoff)).WithJitter(0.2)
```

We should probably also be updating in batches